packages/account/src/Components/address-details/address-details.tsx

```diff
@@ -74,13 +124,13 @@ const AddressDetails = ({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
-    const is_submit_disabled_ref = React.useRef(true);
+    const is_submit_disabled_ref = React.useRef<boolean | undefined>(true);
```

Comment:
since it has a default value this `undefined` doesn't make sense.
```suggestion
    const is_submit_disabled_ref = React.useRef<boolean>(true);
```